lib/cerebrum/cerebrum.rb

```diff
@@ -138,9 +136,10 @@ def calculate_deltas(target)
 
   def run_input(input)
     @outputs[0] = input
+    output = []
```

Comment:
look at the diff generated, this was not merged correctly. `output = []` was removed, this is no longer the return statement but `@outputs.last` is which was moved down to [here](https://github.com/irfansharif/cerebrum/pull/10/files#diff-fd735047d6442f7f9ab1c85395a69b3cR205) for you.